AElf.Contracts.Consensus.DPoS.Tests/ContractTesterExtensions.cs

```diff
@@ -671,5 +671,44 @@ public static async Task<long> GetLIBOffset(this ContractTester<DPoSContractTest
         }
 
         #endregion
+
+        #region  Others
+        
+        public struct User
+        {
+            public ECKeyPair KeyPair { get; set; }
+            public Address Address { get; set; }
+            public string PublicKey { get; set; }
+
+            public static implicit operator ECKeyPair(User user)
+            {
+                return user.KeyPair;
+            }
+
+            public static implicit operator Address(User user)
+            {
+                return user.Address;
+            }
+
+            public static implicit operator string(User user)
+            {
+                return user.PublicKey;
+            }
+        }
+        public static User GenerateNewUser(this ContractTester<DPoSContractTestAElfModule> starter)
```

Comment:
starter is not used.

it is tester not starter

Reply:
renamed